src/org/labkey/test/util/PermissionsHelper.java

```diff
@@ -33,42 +34,46 @@
  */
 public abstract class PermissionsHelper
 {
-    private static final String permissionClassPkg = "org.labkey.api.security.roles.";
-    public String toRole(final String perm)
+    public static final String SITE_ADMIN_ROLE = "Site Administrator";
+    public static final String APP_ADMIN_ROLE = "Application Admin";
+    public static final String DEVELOPER_ROLE = "Platform Developer";
+    public static final String IMP_TROUBLESHOOTER_ROLE = "Impersonating Troubleshooter";
+
+    public static String toRole(final String name)
```

Comment:
Should callers use the above constants for common roles instead of hard-coded strings?